### DISCUSSION TITLE
Scrolling Acceleration

### DISCUSSION THREAD

**Discussion by jrmoulton** [wezterm/wezterm#3191] [Q&A]
Does wezterm have scrolling acceleration as a configuration option? I couldn't find anything in the docs. This is one feature that I've missed from kitty.

**Comment by nekowinston** [comment]
Would something like this help?

```lua
return {
  mouse_bindings = {
    {
      event = { Down = { streak = 1, button = { WheelUp = 1 } } },
      mods = "NONE",
      action = wezterm.action.ScrollByLine(5),
    },
    {
      event = { Down = { streak = 1, button = { WheelDown = 1 } } },
      mods = "NONE",
      action = wezterm.action.ScrollByLine(-5),
    },
  }
}
```
That scrolls the term by 5 lines for each WheelDown/WheelUp trigger. [Here](https://wezfurlong.org/wezterm/config/lua/keyassignment/ScrollByCurrentEventWheelDelta.html)'s the default, I've only tried this quickly on my end, but it seems to work.
